src/main/resources/templates/index.html

```diff
@@ -6,44 +6,179 @@
     <title>Offender Events UI</title>
     <script type="text/javascript">
         function resetFilter() {
-            document.getElementById("event-type-filter").value = "";
+            includeAllEventTypes()
+            excludeNoEventTypes()
             document.getElementById("text-filter").value = "";
             document.getElementById("filter-form").submit();
         }
+
+        function includeAllEventTypes() {
```

Comment:
The sum of all this JavaScript is that you can either select one or more event types to include, or one or more event types to exclude.  By default you include ALL and exclude NONE.  To understand it you'd really have to run the project locally by following the README and running `create-balance-update-messages-local.bash` to inject some messages to filter on.